tornadovmInstaller.sh

```diff
@@ -265,10 +265,18 @@ function setupTornadoVM() {
 }
 
 function resolveBackends() {
-    if [[ "$opencl" && "$ptx" ]] ; then
-	backend="BACKEND=opencl,ptx"
+    if [[ "$opencl" && "$ptx" && "$spirv" ]] ; then
```

Comment:
Why don't we pass directly $1, $2, etc? or `$@`? 

Something like 
`backend="BACKEND=$@"`

Reply:
The order that the user passes a specific `jdk` version or a particular `backend` is arbitrary. So, I am not sure if passing `$1`, `$2` will be valid.

Also, if we pass `$@` this can result in `BACKEND=--jdk8 --ptx --opencl`, right? 
So, in this case it will break.

Reply:
I just saw how this function is reached. At this point is simpler to do the check. 
But I think we can append:

Something like:
```bash
b=""
if $opencl then
   b="$bopencl,"

if $ptx then
   b="$ptx,"

#  remove last comma
... 

backend="BACKEND=$b" 
....